acceptance/features/edit_branch_page_spec.rb

```diff
@@ -45,6 +45,8 @@
     then_I_can_add_and_delete_conditionals_and_expressions
 
     when_I_save_my_changes
+    then_I_should_be_on_the_correct_branch_page('edit')
```

Comment:
@njseeto does this pass for you locally? coz it's currently commented out on line 43

Reply:
Yep this is passing for me locally, I guess that bug has been fixed?

Reply:
only if when line 42 and 43 are uncommented it passes. is that the case if you uncomment them?

Reply:
ok, so when I uncommented ln 42 and 43, they do not pass so the bug must not be fixed yet.
And when I leave ln 42 and 43 commented, but run the test with ln 48, it passes... 🤷 Does it pass for you locally?